### DISCUSSION TITLE
Turn the volume down a bit on map end screen for better compatibility for skins

### DISCUSSION THREAD

**Discussion by ziomek64** [ppy/osu#17544] [Ideas]
So, a lot of map end but the music goes on. I have a skin with music at the end and I can't hear anything at all because the map music plays too loud. In stable osu it was turned down a little bit I believe. In Lazer it is at the same volume as when playing the map. It should be turned down a little or a new setting in sound section should be made. Maybe it's not much but a lot of people use skins like that. Would be great if they were  compatible, well they are but you can't hear much

**Comment by peppy** [comment]
There's no volume adjustment on stable. What you may be noticing is the fact that stable is exceeding system volume limits, with windows automatically dropping the volume levels and potentially making your skin more audible, but this is an unintended side-effect.

We have better headroom in lazer so this may not be happening in the same way.

If possible can you record what you're hearing/seeing?

**Comment by ziomek64** [comment]
I don't know what is happening on stable but the map song is 80% of the volume on the end than what it is when playing. I can definitely hear it goes down a bit when going from the map to the end screen

In lazer it stays full volume on what it was when playing.

Maybe it's a bad example as the track is a part of marathon but nonetheless you will hear what I mean. The guitar is the ONLY thing from my skin playing. Other music and singing is not. https://gofile.io/d/9S0ouv  - the two sound are just blended together, which makes it hard to hear my skin playing tbh. If you don't trust me with this link I will find some other way as github does not allow sound files and there is no reason to record video.

**Reply by frenzibyte** [reply]
I'm not sure what you expect to be taken from that audio file alone, we need a comparison between stable and lazer, and better on a video if feasible, to avoid potentially confusing both files.

**Reply by ziomek64** [reply]
Even if we set aside the stable version, do you think it's fine  for two sounds to blended like that together? I can't hear anything

**Reply by frenzibyte** [reply]
Actions cannot be taken based on only one skin and instance. If it's an issue across stable and lazer then please demonstrate that using audio/video clips, or I might look at it myself if you won't bother.

**Comment by ziomek64** [comment]
https://www37.zippyshare.com/v/sQX3m7ja/file.html
Zip file with 4 videos, lazer and stable, with and without ending music skin. Again, it might be bad example as I play one map of marathon. I don't know how many single songs continue to have music on the ending screen, but it shouldn't be issue with the skin You can definitely hear a bit lower volume on end screen in stable and it goes higher when leaving the end screen

**Reply by frenzibyte** [reply]
Converted to mp4s for embedding instead of zip file:

https://user-images.githubusercontent.com/22781491/160949576-38bdb193-fa68-48db-8b2f-7089b8741d97.mp4

https://user-images.githubusercontent.com/22781491/160949596-35ac8090-8ee1-4013-9bea-cb91c994e505.mp4

https://user-images.githubusercontent.com/22781491/160949599-3e784134-34cc-46f8-8657-32c25d8eeced.mp4

https://user-images.githubusercontent.com/22781491/160949800-b2b3fb2e-8828-4f3e-b7e2-3d5350e9f874.mp4

@ziomek64 While at it, can you attach the skins used in the provided video recordings?

**Reply by frenzibyte** [reply]
On a quick look via audacity, it does seem like there's somewhat of a down-tuning on the track on stable:
<img width="267" alt="CleanShot 2022-03-31 at 03 16 08@2x" src="https://user-images.githubusercontent.com/22781491/160951335-867fbe7a-ac9c-48af-91f1-ced674c0632c.png">

Assuming you have the same exact volume settings set across stable and lazer, this seems to mostly aligns with what @peppy has mentioned in https://github.com/ppy/osu/discussions/17544#discussioncomment-2467373.

**Comment by AadilRn** [comment]
I'm gonna be honest, I could barely tell a difference between stable and lazer here.

**Comment by ziomek64** [comment]
To be honest I'm hearing this small fall of volume in stable. It makes the difference for me. I don't exactly know how to understand peppy comment but I hear it. We could push it even further or make a setting out of this. If it's an open source project it would be nice to tune the game to most of the players. 

And yes I had same volume settings on both

Skins - https://www53.zippyshare.com/v/Es4IRVhS/file.html
